web_responsive/__manifest__.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2017-2018 Tecnativa - Jairo Llopis
+# Copyright 2018-2019 Tecnativa - Alexandre Díaz
+# Copyright 2021 ITerra - Sergey Shebanin
+# Copyright 2023 Onestein - Anjeel Haria
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Web Responsive",
+    "summary": "Responsive web client, community-supported",
+    "version": "16.0.1.0.0",
+    "category": "Website",
+    "website": "https://github.com/OCA/web",
+    "author": "LasLabs, Tecnativa, ITerra, Onestein, "
+    "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "installable": True,
+    "depends": ["web", "mail"],
+    "development_status": "Production/Stable",
+    "maintainers": ["Yajo", "Tardo", "SplashS"],
+    "excludes": ["web_enterprise"],
+    "data": ["views/web.xml"],
+    "assets": {
+        "web.assets_frontend": [
+            # "/web_responsive/static/src/legacy/js/website_apps_menu.js",
+            # "/web_responsive/static/src/legacy/scss/website_apps_menu.scss",
+        ],
+        "web.assets_backend": [
+            "/web_responsive/static/src/legacy/scss/web_responsive.scss",
+            "/web_responsive/static/src/legacy/js/web_responsive.js",
+            # "/web_responsive/static/src/legacy/scss/kanban_view_mobile.scss",
```

Comment:
Can you remove the out commented assets here, please.

Reply:
Done